analysis/study_definition.py

```diff
@@ -1,168 +1,376 @@
+from datetime import date
 
-# Import functions
+from cohortextractor import StudyDefinition, patients
+import codelists
 
-from cohortextractor import (
-    StudyDefinition, 
-    patients, 
-    codelist, 
-    codelist_from_csv
-)
-
-# Import codelists
-
-chronic_cardiac_disease_codes = codelist_from_csv(
-    "codelists/opensafely-chronic-cardiac-disease.csv", system="ctv3", column="CTV3ID"
-)
-chronic_liver_disease_codes = codelist_from_csv(
-    "codelists/opensafely-chronic-liver-disease.csv", system="ctv3", column="CTV3ID"
-)
-salbutamol_codes = codelist_from_csv(
-    "codelists/opensafely-asthma-inhaler-salbutamol-medication.csv",
-    system="snomed",
-    column="id",
-)
-systolic_blood_pressure_codes = codelist(["2469."], system="ctv3")
-diastolic_blood_pressure_codes = codelist(["246A."], system="ctv3")
-
-
-# Specifiy study defeinition
 
 study = StudyDefinition(
-    # Configure the expectations framework
     default_expectations={
-        "date": {"earliest": "1900-01-01", "latest": "today"},
-        "rate": "exponential_increase",
+        "date": {"earliest": "1970-01-01", "latest": "today"},
+        "rate": "uniform",
+        "incidence": 0.1,
     },
-    # This line defines the study population
-    population=patients.registered_with_one_practice_between(
-        "2019-02-01", "2020-02-01"
-    ),
 
-    # https://github.com/opensafely/risk-factors-research/issues/49
+    population=patients.all(),
+
     age=patients.age_as_of(
-        "2020-02-01",
+        "2021-01-01",
         return_expectations={
-            "rate": "universal",
-            "int": {"distribution": "population_ages"},
+            "int": {'distribution': 'normal', 'mean': 40, 'stddev': 20},
         },
     ),
-    
-    # https://github.com/opensafely/risk-factors-research/issues/46
+
     sex=patients.sex(
         return_expectations={
-            "rate": "universal",
-            "category": {"ratios": {"M": 0.49, "F": 0.51}},
-        }
+            "category": {'ratios': {'F': 0.495, 'M': 0.495, '': 0.01}},
```

Comment:
Again minor but the real data would have small numbers of 'U' and 'I' rather than blank (for adults more 'I' than 'U')